test/allowlist_for_publicAPI.json

```diff
@@ -1469,6 +1469,7 @@
     "no_observer_set",
     "prepare",
     "prepare_dynamic_jit",
+    "prepare_ondevice_dynamic_jit",
```

Comment:
Nothing should be added to these lists.
You should make sure to fix the corresponding submodule.

Reply:
@albanD Meaning? Add `__moduel__` tag to the API?

Reply:
I will actually just prefix this with `_` since I dont envision us having to support this for long term.

Reply:
if they should just be private, prefixing with `_` will work indeed.